_posts/2021-01-10-boa-release-11.md

```diff
@@ -0,0 +1,66 @@
+
+While Regress is not 100% spec compliant this is something which is being worked on, also the switch gave us quite a performance boost in our [benchmarks](https://boa-dev.github.io/boa/dev/bench/) we're seeing almost 6X faster execution.
+
+<picture>
+    <source srcset="/images/2021-09-11/regex-bench-dark.png" media="(prefers-color-scheme: dark)">
+    <img src="/images/2021-09-11/regex-bench-white.png">
+</picture>
+
+The above image shows a big drop in the middle of the graph, above `fb1b8d5` is where we switched over. Conformance went from 19.01% to 18.99% and introduced some panics, however many of those have since been fixed.
+
+## Iterating over bytes
+
+Previously the lexer iterated over Unicode chars (u32 code points), this wasn't strictly neccesary for Boa and we have instead changed the lexer to work over bytes (u8). Iterating over bytes rather than chars is inherently much faster, non-ascii UTF8 bytes are all >=128, and we might only really care about those being correct when parsing idents. This is standard practise amongs lexical analyzers and even browsers read source code byte-by-byte, using the rules of the language's syntax to convert the source text into tokens.
+
+This was worked on by @jevancc and his [changes](https://github.com/boa-dev/boa/pull/915/files) have improved performance overall.
+
+## Embedding examples
+
+We are still working on what the public API should look like, some of these decisions are driven by feedback and the constant changing way which Boa works.  
+Not only you can run Boa against javascript today you can also embed your own objects and classes into the engine before it's ran.
+
+We've offered an [example](https://github.com/boa-dev/boa/blob/master/boa/examples/classes.rs) to show how a class can be constructed in Rust then added to the environment before executing your scripts.
+This should offer a great insight into how you can interop Rust with JavaScript by using Boa.
+
+If there are any examples you would like added, please raise an issue on the main repository.
+
+## Generating bytecode
+
+Today Boa walks the tree of the AST, although easy to implement it's not ideal for performance.  
+We are loooking to do code generation which can then be interpreted. This gradual process may happen over many releases until at some point we can switch implementations under the hood. Our steps are:
+
+- ~~Experiment with VM Path on isolated branch~~
+- ~~Experiment with generating more simpler instructions~~
+- Build up parity with current implementation
+- Run test suite over VM path including conformance tests
+- Switch over once performance is steady
+- Optimize code generation and intreperter
+
+## Thank You
+
+Everything in this release has been such a huge effort, we thank everyone who contributed, whether it was features, fixes or raising bugs.
```

Comment:
Here I would mention the [contributors](https://github.com/boa-dev/boa/graphs/contributors?from=2020-10-02&to=2021-01-12&type=c) since the last release. Especially for the ones outside of the org, and the new memberships in the organization.

I would also add links to the [good first issues](https://github.com/boa-dev/boa/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) and maybe also [issues where help is wanted](https://github.com/boa-dev/boa/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22), so that potential contributors know where to start.